compose/compose.go

```diff
@@ -116,6 +136,36 @@ func (c *Compose) MustKill() {
 	}
 }
 
+// GetPublicIPAddressForService returns the IPAddress of a service
+func (c *Compose) GetPublicIPAddressForService(serviceName string) (bool, string) {
```

Comment:
I'll tell you what I was thinking -

_InferDockerHost_ gives you the IPAddress for the machine running the Docker daemon - so for me on Ubuntu it is localhost but if I was running a Mac and using the Docker toolbox (or Docker Machine or whatever it is called now) it would be the IPAddress of the virtual machine running the Docker daemon.

_GetPublicIPAddressForService_ should give you the IPAddress the docker daemon has allocated for the container for the service defined in the docker-compose file.  

_InferDockerHost_ is really useful if your docker-compose file maps container ports to "where ever the docker daemon is running". When you are running simultaneous copies of the docker-compose file you really need to know the IPAddress of the container to call into from your test and that was why I added the method.